Corzbank.Helpers/GetValuesForExchange.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Text;
+
+namespace Corzbank.Helpers
+{
+    public static class GetValuesForExchange
+    {
+        public static dynamic GetValues()
```

Comment:
you should parse that json into your model instead of dynamic usage

Reply:
resolved